apps/web/src/actions/documents/updateContent.ts

```diff
@@ -26,7 +26,7 @@ export const updateDocumentContentAction = withProject
       .then((r) => r.unwrap())
     const docsScope = new DocumentVersionsRepository(ctx.project.workspaceId)
     const document = await docsScope
-      .getDocumentByUuid({
+      .getDocumentAtCommit({
```

Comment:
this was failing for me without this change

Reply:
@csansoon already fixed this